src/resolvers/coordinates.ts

```diff
@@ -0,0 +1,54 @@
+// import { log } from '.';
+// import { UserInputError } from 'apollo-server';
+import { Context } from 'apollo-server-core';
+import { getCoordinates } from '../calculateCoordinates';
+import conf from '../conf';
+import { GraphQLResolveInfo } from 'graphql';
+
+// Resolvers define the technique for fetching the types defined in the
+// schema.
+// I read a doc, where each resolver making a different async call would be better compared
+// to making one single async call
+// because failing to get longitude should not impact latitude
+// I am taking first address address[0], based on the assumption full address is given
+// happy to extend to provide multiple addresses by mappiing through addresses array
+export const resolvers = {
+  Query: {
+    address: (
+      _parent: any,
+      args: { id: string },
+      _context: Context,
+      _info?: GraphQLResolveInfo
+    ) => ({
+      id: args.id,
+    }),
+  },
+  Address: {
+    longitude: async ({ id }: { id: string }) => {
+      try {
+        const address = await getCoordinates(id, conf.apiKey);
+        const longitude = address[0].longitude;
+        return longitude;
+      } catch (err) {
+        // log.logger.error({
```

Comment:
Why are logs commented out?

Reply:
I couldn't make logs working :(, when I used Pino logger i have got errors, i have included more details in `Logger` section in `TechStack.md`, 
what loggers you use at Superformula - happy to give a try.
On a side note, used loggingPlugin - for providing more debug details.

Reply:
Simple console.log statements are a decent starting point if you don't have a proper library to format them. I personally like [lambda-log](https://www.npmjs.com/package/lambda-log) since it's a lightweight and straightforward way to format logs for Cloudwatch

Reply:
wow, i googled a lot about `logger plugin for GraphQL` this `npm install lambda-log` seems pretty straightforward, can give a try today

Reply:
Pushed a commit - using lambda-log, thanks for the info

Reply:
Nice, able to see logs in lambda
<img width="832" alt="Screen Shot 2022-08-16 at 2 05 50 PM" src="https://user-images.githubusercontent.com/22843777/184948598-ca800042-e815-4061-bc05-0a87d302898b.png">